### PR TITLE
Alternate RegEx for errors

### PR DESCRIPTION
**Pull request by quentinrossetti**
Based on the fork of ccrvs.
TODO:
- check if the new regex break compatibility (tests suggest it did)
- why is there a different format for error? (OS, version of 7-Zip?)
- update tests

### CODE REVIEW

**Comment by quentinrossetti** [comment]
Note that tests gives false negative due to early stop.

**Comment by ccsrvs** [comment]
So I didn't realize with Fork it was reaching back to your's and asking to merge in, nor did I receive any notification of your comments. We altered the regex from what we were seeing on mac. As far as testing, we don't use the same testing framework, use jest vs chai. We also were in a pinch to make this work. 

To answer your version questions (finally), we are using MacOS 10.14.6 and the mac version of 7za was not rendering properly. Did you build for windows when building the regex? I wonder if we'll need to have a mac output vs windows or consolidate our regex.

would it be worth severing the fork?

**Comment by quentinrossetti** [comment]
Hi,
No worries, I made those comment as a reminder for myself. I didn't expect to hear from you, but I'm very glad you're here so we can figure out this thing :smiley: 
I develop and test on MacOS, Windows and Linux and didn't find the bug. 
On my side:
`node -v` v11.9.0
`npm -v` v6.14.8
`7za` 16.02
macOS 10.14.6

File: src/events.js [hunk 1]
```diff
@@ -14,17 +14,25 @@
...
   Err.assign(stream, err)
   debug('error: from child process: %O', err)
   return stream
 }
 
-const onStderrFactory = ({ Err }) => (stream, buffer) => {
-  const err = Err.fromBuffer(buffer)
-  Err.assign(stream, err)
-  debug('error: from stderr: %O', err)
+let errorChunk = [];
```
Comment (quentinrossetti) [inline src/events.js:25]: This broke the factory pattern so it broke the test strategy.

**Pull request closed without merging**